### ISSUE TITLE
Error on trying to start tutorial

### ISSUE THREAD

**Issue by andtheWings** [tidymodels/learntidymodels#20]
## The problem

When I run `learnr::run_tutorial("pca_recipes", package = "learntidymodels")`, I get the error:

```r
Quitting from lines 13-80 (pca_recipes.Rmd) 
Error: package or namespace load failed for 'gradethis':
 .onLoad failed in loadNamespace() for 'gradethis', details:
  call: (function (exercise.cap = "Code", exercise.eval = FALSE, exercise.timelimit = 30, 
  error: unused argument (exercise.error.check.code = "gradethis_error_checker()")
```

### Versions:

R itself is 4.0.4
{learnr} is 0.10.1
{learntidymodels} is 0.0.0.9001

**Comment by juliasilge** [comment]
Do you have the current development version of gradethis, as shown in the [DESCRIPTION](https://github.com/tidymodels/learntidymodels/blob/main/DESCRIPTION) file? 

One way you can install the needed dependencies for a package like this is with:

```r
devtools::install_dev_deps()
```
if you are in the working directory of the package.

**Comment by andtheWings** [comment]
Got it to work now! But just to describe what I did:

1. Ran `devtools::install_dev_deps()` for {learntidymodels} and it installed some other packages, but not {gradethis}. I still got the same error on trying to run `learnr::run_tutorial("pca_recipes", package = "learntidymodels")`.
2. I manually checked and the version number for {gradethis} matched the listed package version on the {gradethis} website (0.2.4.9000).
3. Removed {gradethis} and tried reinstalling with `remotes::install_github("rstudio/gradethis")`. Still got the same error message.
4. Removed {learntidymodels}, {learnr}, and {gradethis}. Sequentially installed:
    1. {learnr} with `install.packages("learnr")`
    2. {gradethis} with `remotes::install_github("rstudio/gradethis"). I got message: "These packages have more recent versions available. It is recommended to update all of them. Which would you like to update?" And I selected `learnr (0.10.1 -> 17d4ea1c5...) [GitHub]`
    3. {learntidymodels} with `devtools::install_github("tidymodels/learntidymodels")`
